extensions/integration-tests/testfmks/src/typings/ref.d.ts

```diff
@@ -0,0 +1,10 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+// /// <reference types='vscode'/>
+/// <reference path='../../../../../src/vs/vscode.d.ts'/>
```

Comment:
remove

Reply:
I assume you meant the '// ///<reference path='....> line. Removed that.

Reply:
removed the whole file when moving out to separate project.

---
In reply to: [285194893](https://github.com/microsoft/azuredatastudio/pull/5499#discussion_r285194893) [](ancestors = 285194893)